cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/PreferencesHttpHandlerTest.java

```diff
@@ -33,6 +33,7 @@
  * Tests for {@link PreferencesHttpHandler}
  */
 public class PreferencesHttpHandlerTest extends AppFabricTestBase {
+
```

Comment:
We don't have test for other preferences endpoints? 
* Namespace level
* App level
* Program level

Can you add them if they are not there currently?